# carried local patch for the upstream mirror
# file: src/emulator.cpp
# original base: ae1539d32ae041135564c4c61a7b46d026f268ce
diff --git a/src/emulator.cpp b/src/emulator.cpp
index 16de67be..008fc77e 100644
--- a/src/emulator.cpp
+++ b/src/emulator.cpp
@@ -570,17 +570,17 @@ void Emulator::Run(std::filesystem::path file, std::vector<std::string> args,
     std::string remote_host = Common::GetRemoteNameFromLink();
     if (Common::g_is_release) {
         if (remote_host == "shadps4-emu" || remote_url.length() == 0) {
-            window_title = fmt::format("shadPS4 v{} | {}", Common::g_version, game_title);
+            window_title = fmt::format("Junmin Lee GR2fork v5.101 - shadPS4 v{} | {}", Common::g_version, game_title);
         } else {
             window_title =
-                fmt::format("shadPS4 {}/v{} | {}", remote_host, Common::g_version, game_title);
+                fmt::format("Junmin Lee GR2fork v5.101 - shadPS4 {}/v{} | {}", remote_host, Common::g_version, game_title);
         }
     } else {
         if (remote_host == "shadps4-emu" || remote_url.length() == 0) {
-            window_title = fmt::format("shadPS4 v{} {} {} | {}", Common::g_version,
+            window_title = fmt::format("Junmin Lee GR2fork v5.101 - shadPS4 v{} {} {} | {}", Common::g_version,
                                        Common::g_scm_branch, Common::g_scm_desc, game_title);
         } else {
-            window_title = fmt::format("shadPS4 v{} {}/{} {} | {}", Common::g_version, remote_host,
+            window_title = fmt::format("Junmin Lee GR2fork v5.101 - shadPS4 v{} {}/{} {} | {}", Common::g_version, remote_host,
                                        Common::g_scm_branch, Common::g_scm_desc, game_title);
         }
     }
